### PR TITLE
[11.0][add][mrp_warehouse_calendar]

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Depends on:
* [x] stock_warehouse_calendar https://github.com/OCA/stock-logistics-warehouse/pull/454

MRP Warehouse Calendar
====================

With this module the manufacturing orders created from procurements consider
the calendar assigned to the warehouse of the picking type of the
manufacturing order to determine, based on the product's manufacturing
lead time, the planned start date of the manufacturing order.

Further manual replannings of start/end date of the manufacturing order
also consider the lead time using the warehouse calendar days.

Configuration
==========
* This module depends on `stock_warehouse_calendar <https://github.com/OCA/stock-logistics-warehouse>`_

* Go to *Settings* and activate the developer mode.

* Go to *Settings > Technical > Resource > Working Time* and define your
  resource calendar.

* Go to *Inventory > Configuration > Warehouse Management > Warehouses*
  and assign the Resource Calendar.

Usage
=====
When a manufacturing order is created out of a procurement evaluation
(from an orderpoint, MTO,...) the calendar is considered in the computation
of the planned start date of the manufacturing order.

For example, if it takes 1 day to manufacture a product and it is required
for Monday, the manufacturing order will be created with planned start date
on the previous Friday, if the warehouse operates under a Mo-Fri working
calendar.

### CODE REVIEW

File: mrp_warehouse_calendar/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (MiquelRForgeFlow) [inline mrp_warehouse_calendar/__init__.py]: remove line

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by MiquelRForgeFlow — APPROVED** [review]
Should it still be in Beta status?

**Comment by LoisRForgeFlow** [comment]
@mreficent I think so, it's not mature already.

**Pull request merged**